src/lib/Components/Video.tsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from "prop-types"
+import React from "react"
+import { Dimensions, requireNativeComponent, ViewProperties } from "react-native"
+
+const { width } = Dimensions.get("window")
+
+interface Props extends ViewProperties {
+  // FIXME: Remove ?
+  source?: {
+    uri: string
+  }
+  size?: {
+    width: number
+    height: number
+  }
+}
+
+export class Video extends React.Component<Props> {
+  static propTypes = {
+    source: PropTypes.shape({
+      uri: PropTypes.string,
+    }),
+    size: PropTypes.shape({
+      width: PropTypes.number,
+      height: PropTypes.number,
+    }),
+  }
+
+  render() {
+    return (
+      <NativeVideo
+        source={{ uri: "https://s3.amazonaws.com/artsy-vanity-files-staging/BN-Banner_XS_compressed.mp4" }}
+        size={{
+          width,
+          height: 220,
+        }}
```

Comment:
@orta - Since we're using `CGRect` on the native side I'm not sure how to pass in a `width={...}` and `height={...}` as one would more conventionally see in React UI code. Thoughts?

Reply:
I'd expect the shape of the object to transfer for a CGRect to be `{ origin: { x: num, y: num }, size: { width: num, height: num } }` as that's the shape of the C struct in memory